PullRequestClosed/PullRequestClosed.test.ts

```diff
@@ -0,0 +1,112 @@
+import * as github from '@actions/github';
+import { PullRequestClosed } from './PullRequestClosed';
+import { LogTester } from '../tests/LogTester';
+import { jiraClientStub } from '../tests/JiraClientStub';
+import { createOctokitRestStub } from '../tests/OctokitRestStub';
+
+
+async function runAction(jiraProject: string, title: string, body?: string, user: string = 'test-user') {
```

Comment:
Never set => doesn't need to be here
```suggestion
async function runAction(jiraProject: string, title: string, user: string = 'test-user') {
```